web/src/styles.css

```
@@ -0,0 +1,877 @@
+  --font-success-detail: 400 var(--font-size-success-detail) / var(--line-height-body)
+    var(--font-family-body);
+  --line-height-body: 1.5;
+  --line-height-editor: 1.55;
+  --line-height-tight: 1;
+  --line-height-title: 0.92;
+
+  background: var(--color-page);
+  color: var(--color-text);
+  font: var(--font-body);
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.9.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)